src/naive/date/tests.rs

```diff
@@ -493,24 +493,27 @@ fn test_date_sub() {
 
 #[test]
 fn test_date_add_days() {
-    fn check((y1, m1, d1): (i32, u32, u32), rhs: Days, ymd: Option<(i32, u32, u32)>) {
-        let lhs = NaiveDate::from_ymd(y1, m1, d1).unwrap();
-        let sum = ymd.map(|(y, m, d)| NaiveDate::from_ymd(y, m, d).unwrap());
-        assert_eq!(lhs.checked_add_days(rhs), sum);
+    fn check(lhs: Result<NaiveDate, Error>, days: Days, rhs: Result<NaiveDate, Error>) {
```

Comment:
Is this a change that would be nice to backport to the main branch (to keep the code similar)?

Reply:
Can you check e3fc29b073e2f8b195c79d0570403c96ed0c5e7a if this is what you have in mind? Should I open a pull request against `main`?

Reply:
Nice, yes please.